src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,9 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+
+       if ( (a >= b) && (b >= c ) || (a <= b) && (b <= c))  return true;
+
+        else return false;
```

Comment:
You don't need `if` statement here, cause any comparison in Java will return boolean value for you.
So here you can just return result of your comparisons.